readthedocs/core/templates/core/dns-404.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{% load core_tags %}
+{% load i18n %}
+
+{% block title %}
+  {% trans "Maze Found - Invalid Host" %}
+{% endblock %}
+
+{% block header-wrapper %}
+  {% include "error_header.html" %}
+{% endblock %}
+
+{% block notify %}{% endblock %}
+
+{# Hide the language select form so we don't set a CSRF cookie #}
+{% block language-select-form %}{% endblock %}
+
+
+{% block content %}
+<h1>{% trans '404 - Invalid Host' %}</h1>
+<h3>{% blocktrans %}The host "{{ host }}" is unknown to Read the Docs{% endblocktrans %}</h3>
+
+<p>
+  {% with docsurl='https://docs.readthedocs.io/en/stable/custom_domains.html' %}
+  {% blocktrans trimmed %}
+    If you control this domain and believe this is in error,
```

Comment:
Shouldn't be `is an error`?

```suggestion
    If you control this domain and believe this is an error,
```

Reply:
Either works but I think yours is better.